docs/services/slam/cloudslam/_index.md

```diff
@@ -0,0 +1,422 @@
+
+This example JSON configuration:
+
+- adds the `viam:rplidar`, `viam:cartographer`, and `viam:cloudslam-wrapper` modules
+- configures the `viam:slam:cartographer`, `viam:cloudslam-wrapper:cloudslam`, and the [data management](/services/data/) services
+- adds a `viam:lidar:rplidar` camera with data capture configured
+
+  <br>
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "rplidar",
+      "namespace": "rdk",
+      "type": "camera",
+      "model": "viam:lidar:rplidar",
+      "attributes": {},
+      "service_configs": [
+        {
+          "type": "data_manager",
+          "attributes": {
+            "capture_methods": [
+              {
+                "method": "NextPointCloud",
+                "capture_frequency_hz": 5,
+                "additional_params": {}
+              }
+            ]
+          }
+        }
+      ]
+    }
+  ],
+  "services": [
+    {
+      "name": "carto",
+      "namespace": "rdk",
+      "type": "slam",
+      "model": "viam:slam:cartographer",
+      "attributes": {
+        "enable_mapping": true,
+        "use_cloud_slam": true,
+        "existing_map": "",
+        "camera": {
+          "data_frequency_hz": "5",
+          "name": "rplidar"
+        }
+      }
+    },
+    {
+      "name": "data_manager-1",
+      "namespace": "rdk",
+      "type": "data_manager",
+      "attributes": {
+        "capture_dir": "",
+        "capture_disabled": false,
+        "sync_disabled": false,
+        "tags": [],
+        "additional_sync_paths": [],
+        "sync_interval_mins": 0.1
+      }
+    },
+    {
+      "name": "cloudslam",
+      "namespace": "rdk",
+      "type": "slam",
+      "model": "viam:cloudslam-wrapper:cloudslam",
+      "attributes": {
+        "slam_service": "carto",
+        "api_key": "<location-api-key>",
+        "api_key_id": "<location-api-key-id>",
+        "organization_id": "<organization_id>",
+        "location_id": "<location_id>",
+        "machine_id": "<machine_id>",
+        "machine_part_id": "<machine-part-id>"
+      }
+    }
+  ],
+  "modules": [
+    {
+      "type": "registry",
+      "name": "viam_rplidar",
+      "module_id": "viam:rplidar",
+      "version": "0.1.16"
+    },
+    {
+      "type": "registry",
+      "name": "viam_cartographer",
+      "module_id": "viam:cartographer",
+      "version": "0.3.45"
+    },
+    {
+      "type": "registry",
+      "name": "viam_cloudslam-wrapper",
+      "module_id": "viam:cloudslam-wrapper",
+      "version": "0.0.3"
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+For more information about the configuration attributes, see [Attributes](#attributes).
+
+### Running cloudslam
+
+Navigate to the **CONTROL** tab on your machine's page. A few things should be happening:
+
+- (optional) change the refresh frequency on the `cartographer` card to **Manual Refresh**. Since we want to use cloudslam, we do not need to refresh the underlying SLAM algorithm's map.
```

Comment:
```suggestion
- (optional) change the refresh frequency on the `cartographer` card to **Manual Refresh**. Since you want to use CloudSLAM, you do not need to refresh the underlying SLAM algorithm's map.
```